quick-start/src/test/resources/integration-test-data/create-entity-model.json

```diff
@@ -0,0 +1,86 @@
+{
+  "_scale": 1,
+  "editDescription": false,
+  "editBaseUri": false,
+  "editing": false,
+  "hasDocs": false,
+  "dragging": false,
+  "transform": "translate(10, 115) scale(1)",
+  "filename": null,
+  "hubUi": {
+    "vertices": {},
+    "x": 110,
+    "y": 265,
+    "width": 350,
+    "height": 166
+  },
+  "info": {
+    "baseUri": null,
+    "description": null,
+    "title": "prod-entity",
+    "version": "0.0.1"
+  },
+  "definition": {
+    "description": null,
+    "primaryKey": null,
+    "required": [],
+    "elementRangeIndex": [],
+    "rangeIndex": [],
+    "wordLexicon": [],
+    "pii": [],
+    "properties": [
+      {
+        "showCollation": false,
```

Comment:
this is a minor nit, but where did these properties come from?  The values in here are not from entity services or the data hub model.

Reply:
These properties come from UI to the rest endpoint "/api/current-project/entities/{entityName}". It has the Entity UI data along with the entity information. This data is modeled into EntityModel object in quick-start.

I gathered this data by inspecting the endpoint in the browser network tab.